src/components/attendee/MiniProfile/components/index.ts

```diff
@@ -0,0 +1,12 @@
+import { withProfileById } from "components/hocs/db/withProfileById";
+import { withFallback } from "components/hocs/gate/withFallback";
+import { compose } from "lodash/fp";
+
+import { Loading } from "components/molecules/Loading";
+
+import { MiniProfileModal as _MiniProfileModal } from "./MiniProfileModal";
+
+export const MiniProfileModal = compose(
+  withProfileById,
+  withFallback(["isProfileByIdLoaded"], Loading)
+)(_MiniProfileModal);
```

Comment:
Please try to rework this like https://github.com/sparkletown/sparkle/blob/redesign-2021/src/components/templates/ConversationSpace/ConversationSpace.hoc.tsx 

The only difference is I'd not bother with doing interface extension and exclusion. Just a simple one would be good enough.


It will be easier for following/debugging the logic and for possible future Storyboard display of MoniProfileModal

Reply:
created a hoc here: https://github.com/sparkletown/sparkle/pull/2953/commits/a52527d7fce74b5f15447723260cc7715c8614b3